.github/workflows/eks.yaml

```diff
@@ -54,10 +54,10 @@ jobs:
       hosted_provider: eks
       rancher_version: ${{ inputs.rancher_version || '2.8-head' }}
       k3s_version: ${{ inputs.k3s_version || 'v1.27.9+k3s1' }}
-      operator_nightly_chart: ${{ inputs.operator_nightly_chart == true || true }}
-      run_p0_provisioning_tests: ${{ inputs.run_p0_provisioning_tests == true || github.event_name == 'schedule' && true }}
-      run_p0_importing_tests: ${{ inputs.run_p0_imorting_tests == true || github.event_name == 'schedule' && true }}
-      run_support_matrix_provisioning_tests: ${{ inputs.run_support_matrix_provisioning_tests == true || github.event_name == 'schedule' && false }}
-      run_support_matrix_importing_tests: ${{ inputs.run_support_matrix_importing_tests == true || github.event_name == 'schedule' && false }}
-      destroy_runner: ${{ inputs.destroy_runner == true || github.event_name == 'schedule' && true }}
+      operator_nightly_chart: ${{ inputs.operator_nightly_chart == true || (github.event_name == 'schedule' && true) }}
+      run_p0_provisioning_tests: ${{ inputs.run_p0_provisioning_tests == true || (github.event_name == 'schedule' && true) }}
+      run_p0_importing_tests: ${{ inputs.run_p0_importing_tests == true || (github.event_name == 'schedule' && true) }}
+      run_support_matrix_provisioning_tests: ${{ inputs.run_support_matrix_provisioning_tests == true || (github.event_name == 'schedule' && false) }}
```

Comment:
https://github.com/cpinjani/hosted-providers-e2e/actions/runs/7582477410/job/20652154509

```

 Inputs
    hosted_provider: eks
    rancher_version: 2.8-head
    k3s_version: v1.27.9+k3s1
    operator_nightly_chart: false
    run_p0_provisioning_tests: true
    run_p0_importing_tests: false
    run_support_matrix_provisioning_tests: true
    run_support_matrix_importing_tests: false
    destroy_runner: true
    runner_template: hosted-prov-e2e-ci-runner-spot-n2-highmem-16-gl-template-v1
```

This is running provisioning and support matrix provisioning tests. It should be running provisioning and import tests, no?

Reply:
Actually no, ran only provisioning tests while testing the changes (updated PR description)